libs/@guardian/ab-core/src/@types/index.ts

```diff
@@ -23,9 +23,9 @@ export type CoreAPI = {
 };
 
 export type OphanAPIConfig = {
-	serverSideTests?: ServerSideTests;
-	errorReporter?: ErrorReporterFunc;
-	ophanRecord?: OphanRecordFunction;
+	serverSideTests: ServerSideTests;
```

Comment:
Would we want to keep this one as optional? Feels like less of an issue if the object wasn't passed in?

Reply:
We can always make it optional in the future without requiring a major/breaking change.

I think that this library is meant to capture all tests, and part of the reason we did not notice that it had broken was because it’s currently not responsible for registering the tests in `dotcom-rendering`. I think this change will make things more explicit, and can be trivially bypassed with an empty object `{}`.